adyenv6b2ccheckoutaddon/acceleratoraddon/web/src/com/adyen/v6/utils/SameSiteCookieAttributeAppenderUtils.java

```diff
@@ -0,0 +1,134 @@
+/*
+ *                       ######
+ *                       ######
+ * ############    ####( ######  #####. ######  ############   ############
+ * #############  #####( ######  #####. ######  #############  #############
+ *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+ * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
+ *
+ * Adyen Hybris Extension
+ *
+ * Copyright (c) 2020 Adyen B.V.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ */
+
+package com.adyen.v6.utils;
+
+import com.google.common.net.HttpHeaders;
+import de.hybris.platform.servicelayer.config.ConfigurationService;
+import org.apache.commons.collections4.CollectionUtils;
+import org.apache.commons.configuration.Configuration;
+import org.apache.log4j.Logger;
+
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/*
+ * This class uses code written by Igor Zarvanskyi and published on https://clutcher.github.io/post/hybris/same_site_login_issue/
+ */
+public class SameSiteCookieAttributeAppenderUtils {
+
+    private static final Logger LOG = Logger.getLogger(SameSiteCookieAttributeAppenderUtils.class);
+
+    private ConfigurationService configurationService;
+
+    private static final String PLATFORM_VERSION_PROPERTY = "build.version.api";
+    private static final String SAMESITE_COOKIE_HANDLER_ENABLED_PROPERTY = "adyen.samesitecookie.handler.enabled";
+    private static final List<String> SAP_VERSIONS_WITH_SAMESITE_FIX = Collections.singletonList("2005");
+    private static final List<String> COOKIES_WITH_FORCE_SAME_SITE_NONE = Arrays.asList("JSESSIONID", "acceleratorSecureGUID");
```

Comment:
how about yacceleratorstorefrontRememberMe?

Reply:
Yes, it's also generated on logins. Added it.